### DISCUSSION TITLE
Bug with handling of bool and false as default value

### DISCUSSION THREAD

**Discussion by johnbrett** [supabase/supabase#2092] [General]
I have the following table: 
![image](https://user-images.githubusercontent.com/4015939/123453014-4e1ed980-d5df-11eb-8f04-b225f2a52d76.png)

When queried in the following way, it seems not to register the "false"'s that were created as a default value, only the one I manually set to false:
![image](https://user-images.githubusercontent.com/4015939/123453151-78709700-d5df-11eb-96c5-66d2c8734cfd.png)

It's as if the values set to false as a default value...aren't really false?

**Comment by johnbrett** [comment]
After running:
```
UPDATE jobs
SET verified = false
WHERE verified IS NOT TRUE;
```
I now get the correct number of rows... but what is this fake false that exists ?

**Reply by awalias** [reply]
Very strange, John. Do the fake falses persist when you refresh the dashboard?
